Dockerfile

```diff
@@ -58,10 +58,20 @@ ENV PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:${PATH}
   XDG_CACHE_HOME=/home/linuxbrew/.cache
 WORKDIR /home/linuxbrew
 
-RUN mkdir -p \
+
+RUN --mount=type=cache,target=/tmp/homebrew-core,uid=1001,sharing=locked \
```

Comment:
Is this to avoid repeatedly recloning homebrew-core if it exists already in `/tmp/homebrew-core` and/or to clone it to this location on disk outside the container?

Reply:
yes, this is what brings the performance improvement since it doesn't need to clone the repo from scratch each time if it's already in the cache

Reply:
is there a way to use a subdirectory of e.g. `TMPDIR` on the host instead?

Reply:
no, not really. This needs to happen during the build process